runtime/cere/src/lib.rs

```diff
@@ -1444,12 +1469,7 @@ pub mod migrations {
 	}
 
 	/// Unreleased migrations. Add new ones here:
-	pub type Unreleased = (
-		pallet_ddc_customers::migration::v2::MigrateToV2<Runtime>,
-		pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
-		pallet_ddc_nodes::migrations::v1::MigrateToV1<Runtime>,
-		UpgradeSessionKeys,
-	);
+	pub type Unreleased = UpgradeSessionKeys;
```

Comment:
You should add ```	pallet_ddc_customers::migration::v2::MigrateToV2<Runtime>,
		pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
		pallet_ddc_nodes::migrations::v1::MigrateToV1<Runtime>,``` to the `Unreleased` in order to not miss them when we are ready to deploy DAC to Testnet/Mainnet.

Reply:
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/4e1e6345-cb98-415d-a9c3-76d23e0af517">